src/test/java/task12/Task.java

```diff
@@ -0,0 +1,35 @@
+package task12;
+
+import java.util.Random;
+import java.util.Scanner;
+public class Task {
+    static void task1(){
+        Random rand = new Random();
+        int[] mass = new int[10];
+        for (int i = 0; i <mass.length;i++){
+            mass[i] = rand.nextInt();
```

Comment:
it would be great, if you would handle a situation when user would enter not expected INTEGER number, but text for example.

Reply:
User doesn`t enter number, because random numbers filling automatically

Reply:
makes sense ;) my bad